scout/parse/variant/genotype.py

```diff
@@ -215,3 +268,39 @@ def parse_genotype(variant, ind, pos):
     gt_call["genotype_quality"] = int(variant.gt_quals[pos])
 
     return gt_call
+
+
+def _parse_format_entry(variant, pos, format_entry_name):
+    """Parse genotype format entry for named integer values.
+    Expects that ref/alt values could be separated by /.
+
+    Args:
+        variant(cyvcf2.Variant)
+        pos(int): individual position in VCF
+        format_entry_name: name of format entry
+    Returns:
+        (ref(int), alt(int)) tuple
+    """
+
+    ref = None
+    alt = None
+    if format_entry_name in variant.FORMAT:
+        try:
+            value = variant.format(format_entry_name)[pos]
+            values = list(value.split("/"))
+
+            ref_value = None
+            alt_value = None
+
+            if len(values) > 1:
+                ref_value = int(values[0])
+                alt_value = int(values[1])
+            if len(values) == 1:
+                alt_value = int(values[0])
+            if ref_value >= 0:
+                ref = ref_value
+            if alt_value >= 0:
+                alt = alt_value
+        except (ValueError, TypeError) as e:
```

Comment:
This wouldn't cause any trouble I guess :)

Reply:
😆 Nooo! How could it ever? NoneType is a mixed blessing..